systems/controllers/iiwa_velocity_broadcaster.cc

```diff
@@ -0,0 +1,140 @@
+
+#include "drake/common/trajectories/piecewise_polynomial.h"
+#include "drake/systems/lcm/lcm_publisher_system.h"
+#include "drake/systems/lcm/lcm_subscriber_system.h"
+#include "drake/lcm/drake_lcm.h"
+#include "drake/systems/analysis/simulator.h"
+#include "drake/systems/framework/diagram.h"
+#include "drake/systems/framework/diagram_builder.h"
+#include "drake/lcmt_iiwa_command.hpp"
+#include "drake/lcmt_iiwa_status.hpp"
+#include "drake/manipulation/util/sim_diagram_builder.h"
+#include "drake/examples/kuka_iiwa_arm/iiwa_lcm.h"
+#include "drake/systems/primitives/trajectory_source.h"
+
+#include "systems/controllers/iiwa_velocity_controller.h"
+#include "systems/controllers/iiwa_position_controller.h"
+
+namespace dairlib {
+
+int do_main(int argc, char* argv[]) {
+
+  drake::lcm::DrakeLcm lcm;
+  drake::systems::DiagramBuilder<double> builder;
+
+  // Creating end effector trajectory
+  // TODO make this modular
+  const std::vector<double> times {0.0, 25.0, 35.0, 45.0, 55.0, 65.0, 75.0, 85.0, 95.0, 105.0, 115};
+
+  std::vector<Eigen::MatrixXd> points(times.size());
+  
+  Eigen::Vector3d AS, A0, A1, A2, A3, A4, A5, A6, A7, A8, A9;
+
+  AS << -0.23, -0.2, 0.05;
```

Comment:
Some comments here would be helpful. At least describe the intent of these waypoints.